include/compiler/glue-compiler.h

```diff
@@ -201,7 +201,7 @@ class GlueCompiler : public glue::GlueCompilerInterface {
     hilti::Expression location(const glue::ExpressionAccessor& e);
 
     Driver* _driver = nullptr; /**< driver provided to Init() */
-    int _zeek_version = -1;    /**< Zeek version provided to Init() */
+    std::optional<int> _zeek_version = -1;    /**< Zeek version provided to Init() */
```

Comment:
The point of switching to `std::optional` is to get rid of the magic value `-1` ;)

```suggestion
std::optional<int> _zeek_version; /**< Zeek version provided to Init() */
```